server/command-line/storage.ts

```diff
@@ -20,6 +21,18 @@ program
 		});
 	});
 
+program
+	.command("clean")
+	.argument("[user]", "clean messages for a specific user only, all if not provided")
```

Comment:
```suggestion
	.argument("[username]", "clean messages for a specific username only, all if not provided")
```

nit: best to be specific

Reply:
agree on the param name, but we do it for a user, not a username. So let's meet in the middle and only convert the first, is that ok?

Reply:
fixed